test/Tests.Infrastructure/RachisConsensusTestBase.cs

```diff
@@ -230,6 +263,19 @@ protected RachisConsensus<CountingStateMachine> SetupServer(bool bootstrap = fal
             return rachis;
         }
 
+        private readonly ConcurrentSet<string> _localPathsToDelete = new ConcurrentSet<string>(StringComparer.OrdinalIgnoreCase);
+
+        protected string NewDataPath([CallerMemberName] string prefix = null, string suffix = null, bool forceCreateDir = false)
+        {
+            if (suffix != null)
+                prefix += suffix;
+            var path = RavenTestHelper.NewDataPath(prefix, 0, forceCreateDir);
+
+            _localPathsToDelete.Add(path);
+
+            return path;
+        }
```

Comment:
don't we have a method like that already?

Reply:
Not in RachisConsensusTestBase